confidential-data-hub/secret/src/bin/secret_cli.rs

```diff
@@ -80,12 +106,57 @@ struct AliProviderArgs {
     client_key_file_path: String,
 }
 
+#[cfg(feature = "ehsm")]
+#[derive(Args)]
+struct EhsmProviderArgs {
+    /// path of the crendential file
+    #[arg(short, long)]
+    crendential_file_path: String,
+
+    /// endpoint of eHSM service
+    #[arg(short, long)]
+    endpoint: String,
+}
+
 #[tokio::main]
 async fn main() {
     let args = Cli::parse();
     match args {
-        Cli::Unseal => {
-            // TODO
+        Cli::Unseal(para) => {
```

Comment:
It might be nice to make a separate function for unseal and seal rather than have main be so large. also, nit: `para` -> `unseal_args`